project_1/include/simulator.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "utils.hpp"
+
+#include <vector>
+#include <cstdint>
+#include <queue>
+
+class NetworkSimulator {
+private:
+    std::uint32_t bandwidth = 52428800;
+    std::uint32_t data_transmission_frequency_bytes = 52428800;
+    std::uint32_t data_transmission_frequency_packages = 51200;
+    std::uint32_t package_size = 1024;
```

Comment:
nit: package -> packet

`package` usually refers to a software package. In the context of networks this is called `packet`.